Dockerfile.centos7

```diff
@@ -180,6 +195,38 @@ RUN cd /tmp && curl -L https://sourceforge.net/projects/zsh/files/zsh/5.5.1/zsh-
     sudo echo "/usr/local/bin/zsh" >> /etc/shells && \
     cd .. && rm -fr zsh-*
 
+
+#######################################
+# go get installs
```

Comment:
SHOULD: consolidate all these go get `RUN`s into a single `RUN` to reduce layers.